### PR TITLE
fix: added comment for using default registry url

### PR DESCRIPTION
**Pull request by hellatan**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1,7 @@
+# being behind a proxy has potentially for appending
```
Comment (nameofname) [inline .npmrc]: This sentence is grammatically off, but overall the comment makes sense to me

Comment (hellatan) [reply .npmrc]: this is why i'm a developer, not an english teacher 😄

Comment (hellatan) [reply .npmrc]: oh but this is bad...i'm going to fix it hahaha

**Comment by hellatan** [comment]
not publishing as this will get picked up in the next bump

**Pull request merged**